packages/@vue/cli/lib/Generator.js

```diff
@@ -181,9 +181,13 @@ module.exports = class Generator {
       'private',
       'description',
       'author',
+      'main',
```

Comment:
While we are at it, there are some other related fields:
```suggestion
      'main',
      'module',
      'js:next',
      'browser',
```